app/views/landing_page/blocks/_hero.html.erb

```diff
@@ -11,10 +11,10 @@
 
 <%= content_tag("div", class: hero_classes) do %>
   <%= picture_tag(class: "app-b-hero__imagewrapper") do %>
-    <%= tag.source srcset: "#{image_path(block.image.sources.desktop)}, #{image_path(block.image.sources.desktop_2x)} 2x", media: "(min-width: 769px)" %>
-    <%= tag.source srcset: "#{image_path(block.image.sources.tablet)}, #{image_path(block.image.sources.tablet_2x)} 2x", media: "(min-width: 641px) and (max-width: 768px)" %>
-    <%= tag.source srcset: "#{image_path(block.image.sources.mobile)}, #{image_path(block.image.sources.mobile_2x)} 2x", media: "(max-width: 640px)" %>
-    <%= image_tag(block.image.sources.desktop, alt: block.image.alt, class: "app-b-hero__image") %>
+    <%= tag.source srcset: "#{block_image_path(block.image.sources.hero_desktop_1x)}, #{block_image_path(block.image.sources.hero_desktop_2x)} 2x", media: "(min-width: 769px)" %>
```

Comment:
Currently [my branch drops the `hero_` prefix](https://github.com/alphagov/whitehall/pull/9579/files#diff-bec4c63fd7b93e9d9e92a09e0efaaa528c3c3ee2c1a09e5b689970a3d870b1f8R158-R192) from the sources hash, so they're just `desktop_2x` etc. That was just because it felt like duplication to be doing `- type: "hero"` and then reiterating `hero_` a bunch of times. But it does introduce an inconsistency with the version names configured in whitehall, as those have to be globally unique, so they do need prefixes.

I don't mind either way really - I'd be happy to update my branch to put back the `hero_`s if you think its more consistent to have them. Or we could update them here.

Reply:
🤔  actually maybe I can get it to match whats currently there exactly, and we can lose this PR 🤔